list_dependencies/deps.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+import subprocess
+import json
+
+def format_list(mylist):
+    if len(mylist) < 2:
+        return str(mylist)[1:-1]
+    else:
+        return str(mylist[:-1])[1:-1] + " and '" + str(mylist[-1]) + "'"
+
+def get_node_info():
+    stdout_str = subprocess.run(['tilt', 'dump', 'engine'], stdout=subprocess.PIPE).stdout.decode('utf-8')
+    data = json.loads(stdout_str)["ManifestTargets"]
+
+    dependencies = {}
+    node_ready = {}
+
+    for resInfo in data.values():
+        manifest = resInfo["Manifest"]
+        resName = manifest["Name"]
+        resDeps = manifest["ResourceDependencies"]
+        if resDeps == None:
+            resDeps = []
+        dependencies[resName] = resDeps
+
+        pods = resInfo["State"]["RuntimeState"]["Pods"]
```

Comment:
similarly, i wonder if we should put ResourceDependencies on the UIResourceStatus type, so you can also read those with `tilt get`. wdyt? 

i think you can only get ResourceDependencies from 'dump engine' right now. but we're generally moving away from 'tilt dump engine' - the problem is that it's one big blob of data, whereas we're moving towards smaller API objects that you can listen to updates on without caring about the others.

Reply:
> can this use 'tilt get kubernetesdiscovery' to fetch the pods? we're much more likely to support that long-term...

Changed. Pod status is now obtained using a separate call to `tilt get kubenetesdiscovery -o json`.

> i wonder if we should put ResourceDependencies on the UIResourceStatus type, so you can also read those with tilt get. wdyt? [...] i think you can only get ResourceDependencies from 'dump engine' right now.

Yes, as far as I can tell, none of the `tilt get` commands' objects currently include dependency information, which is why I used `tilt dump engine` here instead. It would be nice to have dependencies in a new or existing API object.